src/mixins/search-mixin.ts

```diff
@@ -161,6 +161,15 @@ export class SearchMixin extends Mixins(CommonMixin) {
     return XproMapping.CHG.includes(this.getSearchBusiness?.legalType)
   }
 
+  /** Whether company can change name. */
+  get isNameChangeable (): boolean {
+    const corpType = this.getSearchBusiness?.legalType as unknown as CorpTypeCd
+    return (
+      BcMapping.REH.includes(this.corpTypeToEntityType(corpType)) ||
+      XproMapping.REH.includes(this.corpTypeToEntityType(corpType))
+    )
+  }
```

Comment:
"REH" is for restore/reinstate. This is "CHG".

The BcMapping CHG array looks OK but the XproMapping CHG array should not contain XSO.